LocalSettings.php

```diff
@@ -2394,6 +2394,12 @@
 		'wisdomsandboxwiki' => '_blank',
 		'yacresourceswiki' => '_blank',
 	),
+	
+	// Allow External Images
+	'wgAllowExternalImages' => array (
+		'default' => false,
+		'westmarcheswiki' => true
```

Comment:
I see what NDKilla means. I will be making another pullrequest with the fix implemented and I hope doing so is the right method for going about things... If not; I am sorry and new ^^